extensions/parquet-extensions/src/main/java/io/druid/data/input/ParquetHadoopInputRowParser.java

```diff
@@ -0,0 +1,40 @@
+package io.druid.data.input;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import io.druid.data.input.impl.InputRowParser;
+import io.druid.data.input.impl.ParseSpec;
+
+public class ParquetHadoopInputRowParser implements InputRowParser<InputRow>
```

Comment:
i think the type here should be MessageType and all the conversion should happen in this class